compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Util.hs

```diff
@@ -116,6 +116,21 @@ mkIf cond0 then0 else0 =
 mkBool :: Bool -> Expr
 mkBool = EBuiltin . BEBool
 
+pattern ETyApps :: Expr -> [Type] -> Expr
+pattern ETyApps fn args <- (view _ETyApps -> (fn, args))
```

Comment:
I could swear we had those in the past. I wonder if I'm just wrong or if somebody removed them. 🤔